.github/workflows/update-chart.yml

```diff
@@ -0,0 +1,75 @@
+name: Update Helm Chart Version and Create Pull Request
```

Comment:
@andreasisnes You have two choices with the helm chart:

1. Move it completely here. Then you need to also create maybe a $repo_root_folder/charts/altinn-access-management and paste everything inside of the contents from the other repo.
Instead of this update-chart.yaml you just need to add the $repo_root_folder/charts/altinn-access-management to the folders that trigger the build and push workflow.
2. Leave this action as it is but no need to have values.yaml here or anything from the helm chart and just keep it where it is.

Reply:
Or is it that you never do changes in the helm chart except for the values.yaml?

Reply:
we haven't changed that chart since ever. Hehe, we typically just change the values.yaml

Reply:
But there will come some infrastructure changes, so copying the chart can be good idea?

Reply:
Yeah, let's just move the whole thing then (option 1).

Reply:
perfect!